components/admin_console/admin_sidebar/admin_sidebar.tsx

```diff
@@ -229,49 +243,47 @@ class AdminSidebar extends React.PureComponent {
                 });
 
                 // Special case for plugins entries
-                let moreSidebarItems = [];
-                if (section.id === 'plugins') {
-                    moreSidebarItems = this.renderPluginsMenu();
+                if ((section as typeof AdminDefinition['plugins']).id === 'plugins') {
+                    const sidebarPluginItems = this.renderPluginsMenu();
+                    sidebarItems.push(...sidebarPluginItems);
                 }
 
                 // If no visible items, don't display this section
-                if (sidebarItems.length === 0 && moreSidebarItems.length === 0) {
+                if (sidebarItems.length === 0) {
                     return null;
                 }
 
-                if (sidebarItems.length || moreSidebarItems.length) {
-                    sidebarSections.push((
-                        <AdminSidebarCategory
-                            key={key}
-                            definitionKey={key}
-                            parentLink='/admin_console'
-                            icon={section.icon}
-                            sectionClass=''
-                            title={
-                                <FormattedMessage
-                                    id={section.sectionTitle}
-                                    defaultMessage={section.sectionTitleDefault}
-                                />
-                            }
-                        >
-                            {sidebarItems}
-                            {moreSidebarItems}
-                        </AdminSidebarCategory>
-                    ));
-                }
+                sidebarSections.push((
+                    <AdminSidebarCategory
+                        key={key}
+                        definitionKey={key}
+                        parentLink='/admin_console'
+                        icon={section.icon}
+                        sectionClass=''
+                        title={
+                            <FormattedMessage
+                                id={section.sectionTitle}
+                                defaultMessage={section.sectionTitleDefault}
+                            />
+                        }
+                    >
+                        {sidebarItems}
+                    </AdminSidebarCategory>
+                ));
             }
             return null;
         });
         return sidebarSections;
     }
 
-    isPluginPresentInSections = (plugin) => {
-        return this.state.sections && this.state.sections.indexOf(`plugin_${plugin.id}`) !== -1;
+    isPluginPresentInSections = (plugin: PluginRedux) => {
+        return this.state.sections?.indexOf(`plugin_${plugin.id}`) !== -1;
```

Comment:
I am not 100% sure, but this could lead to false results, when `this.state.sections` is `undefined`, since it will return `true` in that case (although pretty unlikely).

```suggestion
        return this.state.sections?.indexOf(`plugin_${plugin.id}`) >= 0;
```